shared/src/dto/user/user.dto.ts

```diff
@@ -78,6 +78,23 @@ export class DtoCreateUser {
   phoneNumber: string;
 }
 
+export class DtoCreateUserSignup {
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsEmail()
+  email: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsBoolean()
+  isActive: boolean;
```

Comment:
Well I don't think the user should be able to specify whether they are active or not 🙂
Also isn't this DTO missing a password property?

Reply:
This was made by someone else before, probably didn't think about fixing the missing thing 🙈